text_file_functions.h

```diff
@@ -0,0 +1,12 @@
+#ifndef TEXT_FILE_FUNCTIONS_H
+#define TEXT_FILE_FUNCTIONS_H
+#endif // TEXT_FILE_FUNCTIONS_H
```

Comment:
From longbyte:

The #endif needs to go at the end of the file in order for the include guard to work.